vendor/CMakeLists.mkbootimg.txt

```diff
@@ -0,0 +1,3 @@
+install(PROGRAMS mkbootimg/mkbootimg.py DESTINATION bin RENAME mkbootimg)
+install(PROGRAMS mkbootimg/unpack_bootimg.py DESTINATION bin RENAME unpack_bootimg)
+install(PROGRAMS mkbootimg/repack_bootimg.py DESTINATION bin RENAME repack_bootimg)
```

Comment:
Are these actually python2 or a python3 scripts? They have a `#!/usr/bin/env python` shebang, if they are indeed python3 scripts that shebang has to be changed to `#!/usr/bin/env python3` otherwise the script will not work on Arch and Alpine.

Reply:
Python defaults to py3 on arch. But that isn't really important, the shebang should be explicit for any script that does not work on both py2 AND py3. (There is a PEP somewhere also saying that /use/bin/python can be either 2 or 3, so you can't expect a specific version)

Reply:
```
head -n1 vendor/mkbootimg/*.py 
==> vendor/mkbootimg/mkbootimg.py <==
#!/usr/bin/env python3

==> vendor/mkbootimg/repack_bootimg.py <==
#!/usr/bin/env python3

==> vendor/mkbootimg/unpack_bootimg.py <==
#!/usr/bin/env python3
```
They are all already with py3 bangs here. Idk why this isn't the case on your system?

Reply:
@nmeum Indeed as @Johnnynator already pointed out the scripts use a `#!/usr/bin/env python3` shebang, [take a look for yourself here](https://android.googlesource.com/platform/system/tools/mkbootimg/+/refs/tags/platform-tools-31.0.3) :)

Reply:
hehe, had an old branch checked out. Sorry for the noise